packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+    <li
+        :class="['c-nav-list-item has-sublist', {
+            'open': countrySelectorIsOpen
+        }]"
+        data-test-id="countrySelector-toggle"
+        v-on="isBelowMid ? null : { mouseover: openCountrySelector, mouseleave: closeCountrySelector }"
+        @keyup.esc="closeCountrySelector">
+        <f-button
+            button-type="icon"
+            class="c-nav-list-text c-country-selector-list-item"
```

Comment:
`c-country-selector-list-item` doesn't really fit the naming atm.

Looking at the current structure, the base list item (line 2) should have a `c-countrySelector` class on it, and then this would be `c-countrySelector-btn` or something I would imagine?

```suggestion
            class="c-nav-list-text c-countrySelector-btn"
```